### DISCUSSION TITLE
Functionality Intergration as part of oneclickinstaller

### DISCUSSION THREAD

**Discussion by tsprasath** [Sunbird-Lern/Community#121] [Installation] [answered]
Lern  consist of 14 API and the following are the observation and resolution found after execution.

1. After catching refresh, the initial attempt at creating an organization resulted in a 400 error.

```
curl --location --request POST 'https://easyinstallgsb.sunbirded.org/api/org/v1/create' \
--header 'Authorization: Bearer ' \
--header 'x-authenticated-user-token: ' \
--header 'Content-Type: application/json' \
--data-raw '{
    "request": {
        "orgName": "Sunbird Org",
        "channel": "sunbird",
        "description": "Default Organisation for sunbird",
        "externalId": "sunbird",
        "isSSOEnabled": true,
        "organisationType": "board",
        "isTenant": true
    }
}'
```
2.Channel Read API was having issues because the KnowledgeMW service was down.
3.Org-create is carried out once more. It must be taken out.
4.To create a user ''/api/user/v1/create'' . the below content should be removed from the body.
```
 "emailVerified": true,
 "channel": "{{channel_name}}",
```
And after that, it was 200 OK.

5. For the content creator and content reviewer, repeat steps 4.

**Comment by Jayaprakash8887** [comment, marked as answer]
Acknowledged. Thank you
